schemas/bolt-defaults.schema.json

```diff
@@ -222,7 +252,7 @@
               "format": "uri"
             },
             "proxy": {
-              "description": "The HTTP proxy to use for Git and Forge operations.",
+              "description": "The HTTP proxy to use for Forge operations.",
```

Comment:
I believe this can still be used for Git modules manually declared in the Puppetfile.

Reply:
This is under `forge` and only applies to Forge operations. The `proxy` setting one level higher is the global setting that applies to both git and Forge operations.

Reply:
Ah ok - sorry, the git diff folded the surrounding lines!